community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/UpdateRecordsStep.java

```diff
@@ -41,23 +44,34 @@
     private final RecordStore<RECORD> store;
     private final int recordSize;
     private int recordsUpdated;
+    private final Predicate<RECORD> updatePredicate;
```

Comment:
Can we give this a more specific name or make the type more specific? I found it quite difficult to understand what was going on when I first read it.